2022-01-01 12:51:05,247 INFO: Initialized curve data.
2022-01-01 12:51:05,250 INFO: Setting up initial twist data...
2022-01-01 12:51:06,384 INFO: Twist setup complete.
2022-01-01 12:51:06,384 INFO: Searching for rational points on each twist...
2022-01-01 12:51:07,305 INFO: Finished searching for rational points on each twist.
2022-01-01 12:51:07,305 INFO: Testing local solvability of twists...
2022-01-01 12:51:07,311 INFO: Finished local solvability testing.
2022-01-01 12:51:07,313 INFO: Computing elliptic curve a-invariants...
2022-01-01 12:51:15,154 INFO: Finished computation of a-invariants.
2022-01-01 12:51:15,157 INFO: Computing MW group (delta = ['28', '-21', '-8', '1', '2', '1'], g = [28, -20, -8, 1, 2, 1])...
2022-01-01 13:02:04,709 INFO: Finished MW computation (delta = ['28', '-21', '-8', '1', '2', '1'], g = [28, -20, -8, 1, 2, 1])
2022-01-01 13:02:04,714 INFO: Computing MW group (delta = ['1', '-1'], g = [28, -20, -8, 1, 2, 1])...
2022-01-01 13:47:26,208 ERROR: An exception occurred. Recording data and exiting.
Traceback (most recent call last):
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 506, in <module>
    rank, MW_proven, orders, gens = twist_MW_group(D)
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 327, in twist_MW_group
    A, mw, bool_rank, bool_index = magma.MordellWeilGroup(E, nvals=_sage_const_4 )
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1787, in __call__
    return M.function_call(self._name,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1162, in function_call
    return self._do_call(fun, nvals)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1221, in _do_call
    raise TypeError("Error executing Magma code:\n%s" % out)
TypeError: Error executing Magma code:

Current total memory usage: 8181.6MB, failed memory request: 32.0MB
System Error: User memory limit has been reached
2022-01-03 13:14:57,000 INFO: Obsolete "loc_solv_error" keys removed.
